packages/react-treat/package.json

```diff
@@ -4,14 +4,16 @@
   "version": "1.1.6",
   "main": "lib/commonjs",
   "module": "lib/module",
+  "types": "lib/types",
   "sideEffects": false,
   "author": "SEEK",
   "license": "MIT",
   "repository": "https://github.com/seek-oss/treat/tree/master/packages/react-treat",
   "scripts": {
-    "build:module": "tsc --module es2015 --outDir lib/module",
-    "build:commonjs": "tsc --module commonjs --outDir lib/commonjs",
-    "build": "concurrently \"yarn build:module\" \"yarn build:commonjs\"",
+    "build:module": "tsc --module es2015 --outDir lib/module --declaration false",
+    "build:commonjs": "tsc --module commonjs --outDir lib/commonjs --declaration false",
+    "build:types": "tsc --emitDeclarationOnly --declaration true --declarationDir lib/types",
+    "build": "concurrently \"yarn:build:module\" \"yarn:build:commonjs\" \"yarn:build:types\"",
     "watch": "concurrently \"tsc --module commonjs --watch\" \"tsc --module module --watch\""
```

Comment:
If we don't update the `watch` command won't the types go stale in watch mode?

Reply:
Sure, but do we care? TypeSctipt is for IDE's, and compile checks. Both of which already happen. Definition files are for consumers - who consume js.